src/github.com/getlantern/golog/golog_test.go

```diff
@@ -112,3 +113,24 @@ func TestAsStdLogger(t *testing.T) {
 	stdlog.Printf("Hello %d", 5)
 	assert.Regexp(t, severitize("ERROR", expectedStdLog), string(out.Bytes()))
 }
+
+func newBuffer() *synchronizedbuffer {
+	return &synchronizedbuffer{orig: &bytes.Buffer{}}
+}
+
+type synchronizedbuffer struct {
+	orig  *bytes.Buffer
+	mutex sync.RWMutex
```

Comment:
Clever idea.  It makes exposes the Mutex api to consumers of synchronizedBuffer, which is neither necessary nor necessarily desirable since it's really just an implementation detail.  So, on the balance I think it's better to stick with mutex being hidden.